Controller/HeadlessWebsiteController.php

```diff
@@ -44,7 +44,7 @@ public function indexAction(
         $json = $this->serializeData($data);
 
         if ('json' !== $request->getRequestFormat()) {
-            return $this->renderTemplateResponse($structure, $requestFormat, $preview, [
+            return $this->renderTemplateResponse($structure, $requestFormat, $preview, $partial, [
```

Comment:
We can replace this call with `renderStructure` if we extend from the `WebsiteController` think this would be the best way to avoid that we need to maintian the partial or preview things.

Reply:
think this would be worth a try. as far as i understand, this would even solve parts of https://github.com/sulu/SuluHeadlessBundle/issues/14. when doing this, there should be the default `content` and `view` variable and the headless `data` and `jsonData` in the twig template

Reply:
maybe we should rename `data` to `headlessArray` and `headlessJson` then 🙂